src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as S from './styles';
+
+interface ButtonProps {
+  type?: 'submit';
```

Comment:
```suggestion
  type?: 'button' | 'submit' | 'reset';
```

버튼이 common이니까 재사용성을 염두에 주신 거 같아서 의견 드립니다.

현재는 'submit`만 받을 수 있어서.. 사실 파라미터로 받는 의미가 없어진 상황이네요.
type을 여러 개 받을 수 있도록 수정하면 그 의미가 살아날 거 같습니다.

그리고 버튼은 주로 click handler가 필요하니까 onClick도 props로 받는 거를 한 번 고려해주세요! (강요 x)
버튼이 항상 form 내부에 있어서 onSubmit으로 제어가 가능하지 않을 수도 있으니까요.